conan/cli/command.py

```diff
@@ -67,6 +68,8 @@ def _init_formatters(self, parser):
         if formatters:
             help_message = "Select the output format: {}".format(", ".join(formatters))
             parser.add_argument('-f', '--format', action=OnceArgument, help=help_message)
+        parser.add_argument("--out-file", action=OnceArgument,
```

Comment:
Missing moving this into the ``if formatters``?

Reply:
Changed to only be present if `self._formatters` that include the default `text` formatter that does not appear in the `--help`.